### DISCUSSION TITLE
When will the new version 0.3.15 be released

### DISCUSSION THREAD

**Discussion by zkk960317** [Ferrite-FEM/Ferrite.jl#821] [Q&A] [answered]
When will the new version 0.3.15 be released

**Comment by KnutAM** [comment, marked as answer]
We are working on releasing 1.0, but there are still a few open things. 
I will work on #692 and #764 in the weekend. Once those are in, I think we are quite close :)
I would guess in the coming weeks unless some new issues pop up
